config/packages/payum.yaml

```diff
@@ -1,17 +1,26 @@
 payum:
     storages:
-        Payum\Core\Model\Payment:
-            filesystem:
-                storage_dir: "%kernel.root_dir%/Resources/payments"
-                id_property: number
+        App\Model\Payment:
+            yadm:
+                id_property: _id
```

Comment:
`_id` is default value, no need to set it explicitly. This should work `App\Model\Payment: { yadm: ~ }`